lib/cdo/graphics/certificate_image.rb

```diff
@@ -48,33 +56,8 @@ def create_course_certificate_image(name, course=nil, sponsor=nil)
       self.fill = 'rgb(118,101,160)' # purple
     end
 
-    # Take course name and convert it to a nicer full course name for the certificate
-    case course
-      when nil
-        full_course_name = ''
-      when 'artist'
-        full_course_name = 'Artist'
-      when 'course1'
-        full_course_name = 'Course 1'
-      when 'course2'
-        full_course_name = 'Course 2'
-      when 'course3'
-        full_course_name = 'Course 3'
-      when 'course4'
-        full_course_name = 'Course 4'
-      when 'frozen'
-        full_course_name = 'Frozen'
```

Comment:
AFAICT this code path was never hit for Hour of Code courses Frozen, Play Lab and Flappy Bird. The fall-back title mapping for one circumstance shows up below w/ a note on why it's needed.